### PR TITLE
Refs #16 Relocates all test files into the tests directory.

### PR DESCRIPTION
**Pull request by don-felipe**
run_tests.bat can be used to launch all test classes contained in the tests
directory.

### CODE REVIEW

File: tests/test_antiweb.py [hunk 1]
```diff
@@ -30,9 +30,10 @@ class Test_Antiweb(unittest.TestCase):
     #This function will be called before each Test. It copies the file to process into the temp directory of the test
     #@code
     def setUp(self):
-        self.directory = "docs"
+        self.tests_dir = os.path.dirname(os.path.abspath(__file__))
+        self.doc_dir = "docs"
         self.test_dir = tempfile.mkdtemp()
-        self.origin_path = os.path.join(os.getcwd(), os.path.join("unittest", "small_testfile.py"))
+        self.origin_path = os.path.join(self.tests_dir, os.path.join("unittest", "small_testfile.py"))
```
Comment (7enderhead) [inline tests/test_antiweb.py]: - we don't need nested join()s:

```
os.path.join(self.tests_dir, "unittest", "small_testfile.py")
```

Comment (7enderhead) [inline tests/test_antiweb.py]: - perhaps we can factor out a function that performs

```
os.path.join(self.tests_dir, "unittest", xyz)
```
- bonus question: why do we need the 'unittest' subdir anyway?

File: tests/test_antiweb.py [hunk 2]
```diff
@@ -145,8 +146,9 @@ def test_antiweb_r(self):
             self.file_not_exist(os.path.join(self.test_dir, "index.rst"))
 
     def tearDown(self):
+        print(self.test_dir)
 
-        shutil.rmtree(self.test_dir)
+        #shutil.rmtree(self.test_dir)
```
Comment (7enderhead) [inline tests/test_antiweb.py]: - temp test dir should be deleted, no?

Comment (don-felipe) [inline tests/test_antiweb.py]: Right.

File: tests/test_antiweb.py (hunk 1 above)
Comment (don-felipe) [inline tests/test_antiweb.py]: - The 'unittest' and 'unittest_rst' folders are not required.
- They help to partition the 'tests' directory into more manageable pieces of data input/output for error tracing (both 'unittest' and 'unittest_rst' contain an index.rst file which would overwrite one another).
- If you take a look into 'tests', it already contains a lot of input files. I personally would put all those files into 'tests/data/...' and create subdirectories for better structuring.

**Comment by don-felipe** [comment]
I created a new pull request because the comments applied to an old commit.

**Pull request closed without merging**